src/components/Card.jsx

```diff
@@ -30,6 +65,12 @@ export default function Card({ chatName, chatLink, from, to, dateFrom, dateTo, m
             </svg>
             <span>{ to }</span>
         </div>
```

Comment:
I don't like it that component is pretty common let's move it to separate file of the component